client/post-editor/editor-title/index.jsx

```diff
@@ -101,19 +104,36 @@ export default React.createClass( {
 				}
 				<TrackInputChanges onNewValue={ this.recordChangeStats }>
 					<TextareaAutosize
-						{ ...omit( this.props, Object.keys( this.constructor.propTypes ) ) }
+						{ ...ownProps }
 						className="editor-title__input"
-						placeholder={ this.translate( 'Title' ) }
+						placeholder={ translate( 'Title' ) }
 						onChange={ this.onChange }
 						onInput={ this.resizeAfterNewlineInput }
 						onBlur={ this.onBlur }
 						autoFocus={ isNew && ! isMobile() }
 						value={ post ? post.title : '' }
-						aria-label={ this.translate( 'Edit title' ) }
+						aria-label={ translate( 'Edit title' ) }
 						ref="titleInput"
 						rows="1" />
 				</TrackInputChanges>
 			</div>
 		);
 	}
-} );
+}
+
+export default connect(
+	( state, ownProps ) => {
+		const site = getSelectedSite( state );
+		const editedPostId = getEditorPostId( state );
+		const post = getEditedPost( state, site.ID, editedPostId );
+		const isNew = isEditorNewPost( state );
+
+		return {
+			isNew,
+			post,
+			site,
+			ownProps
```

Comment:
I'm sorry @youknowriad but I don't think I understand what you are saying here. what it appears like you are saying doesn't fit with what I think the code is saying.

what is the problem with `localize()` and why does it break things? also, one of my chief questions here is that we are forcing the component/children to change their logic here because if they formerly depended on `this.props.value` they will now have to call out to `this.props.ownProps.value`